deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -78,6 +78,7 @@ metadata:
     createdAt: 2021-01-26T09:00:00Z
     support: Infinispan
     certified: "false"
+    olm.skipRange: ">=2.1.0 <2.1.5"
```

Comment:
I'm not sure this makes sense for master as master should be `2.2.0` potentially replacing `2.1.x` (5)? and `2.1.5` should skip `2.1.0 - 2.1.4`. Also looking at the version name of the Operator in line bellow it should skip itself.